src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,46 @@
+import { toggleDrawer } from '../Drawer/utils';
+import { useState } from 'react';
+import { IconButton, Stack, useMediaQuery, Button } from '@mui/material';
+import MenuIcon from '@mui/icons-material/Menu';
+import Drawer from '../Drawer/Drawer';
+import { NavigationT } from '../../types/types';
+
+const Navigation = ({ navigate, siteTreeOptions }: NavigationT) => {
```

Comment:
As mentioned in another comment, `Navigation` only passes navigates down to `Drawer`. Maybe it can be removed and `Drawer` can use the hook to get the `navigate`?

Reply:
✅  I will replace it to the component